zebedee-cms/src/main/java/com/github/onsdigital/zebedee/configuration/Configuration.java

```diff
@@ -196,7 +205,15 @@ public static List<Host> getTheTrainHosts() {
     }
 
     public static List<Host> getWebsiteHosts() {
-        return Arrays.asList(StringUtils.split(defaultIfBlank(getValue("website_url"), DEFAULT_WEBSITE_URL), ","))
+        String defaultLowercaseUrl = "website_url";
+        String defaultUrl = DEFAULT_WEBSITE_URL;
+
+        if (isCacheAPIEnabled()){
+            defaultLowercaseUrl = "cache_api_url";
+            defaultUrl = DEFAULT_CACHE_API_URL;
+        }
+        // check feature if ennbaled here
```

Comment:
Is this a missed TODO? "Enabled" is misspelled, by the way, but I don't think this comment is needed anyway.

Reply:
deleted comment